controller/scan/jfrog.go

```diff
@@ -250,7 +245,7 @@ func (r *jfrog) GetTagList(domain, repo, tag string) ([]string, error) {
 		repo = subRepo
 	}
 
-	if tags, err := rc.Tags(repo); len(tags) > 0 {
+	if tags, err := rc.Tags(repo); len(tags) > 0 || err != nil {
```

Comment:
The only case where you need to fallback is the `len(tags) == 0 && err == nil`?  If so, I recommend leaving a comment.

Reply:
Thanks, I just left comment and interchange the call order, thanks